frontend/express/public/core/user-management/stylesheets/_main.scss

```diff
@@ -0,0 +1,301 @@
+.user-management-view {
```

Comment:
Too much custom CSS, but OK 😄

Reply:
let's find the right way together on DM. :D